ISISComputingGroup/ibex_gui
# Ticket1854: Modified icons to be transparent

### To test

Start the GUI, the taskbar icons will be transparent.

See https://github.com/ISISComputingGroup/IBEX/issues/1854

---

#### Code Review


### Functional Tests


### Final Steps
